base/c9s-python-3.9/Dockerfile

```diff
@@ -1,14 +1,19 @@
 FROM quay.io/sclorg/python-39-c9s:c9s
 
+ARG BUILD_COMMIT_REF
+ARG IMAGE_REGISTRY
+
+ARG AUTHORITATIVE_SOURCE_URL="https://github.com/opendatahub-io/notebooks"
```

Comment:
What if we move also this one outside from the Dockerfile? on the Makefile for example as we do with the other --build-args?

Reply:
yes, agree with you - I pointed out in the point 2 in the description in section `Notable points for discussion` - since you are also favor of it, I'll update my PR this way :slightly_smiling_face:

Reply:
Right! Tnx :)

Reply:
Changed per your proposal :slightly_smiling_face:  ; Also, I removed relevant point from the description since it's irrelevant now.